client/src/Components/Questions.js

```diff
@@ -52,10 +52,8 @@ const Questions = (props) => {
 			body: JSON.stringify(answer),
 		});
 		event.target.reset;
-
-		if (currentQuestionIndex >= props.fetchedData.questions_id.length - 1) {
-			alert("You submited the form successfully!");
-		}
+		props.setIsSubmit(true);
```

Comment:
I would suggest changing the name is to isSubmitted or startSubmit depending on what is happening.it confused me a bit when I first saw this name

Reply:
It's right, I'll change it to isSubmitted